src/commands/distribute/groups/update.ts

```diff
@@ -91,16 +100,32 @@ export default class UpdateDistributionGroupCommand extends AppCommand {
       addedTestersEmails = [];
     }
 
+    if (deletedTestersEmails.length !== testersToDeleteEmails.length || addedTestersEmails.length !== testersToAddEmails.length) {
+      out.text("Updating the list of testers was partially successful");
+    }
+
     let currentGroupName: string;
+    const options: { name?: string; isPublic?: boolean } = {};
     if (!_.isNil(this.newDistributionGroupName)) {
       debug("Renaming the distribution group");
-      currentGroupName = await this.renameDistributionGroup(client, app, this.newDistributionGroupName);
+      options.name = this.newDistributionGroupName;
+      currentGroupName = this.newDistributionGroupName;
     } else {
       currentGroupName = this.distributionGroup;
     }
 
-    if (deletedTestersEmails.length !== testersToDeleteEmails.length || addedTestersEmails.length !== testersToAddEmails.length) {
-      out.text("Updating the list of testers was partially successful");
+    if (this.makePublic) {
+      debug("Setting distribution group public status to true");
+      options.isPublic = true;
+    }
+
+    if (this.makePrivate) {
+      debug("Setting distribution group public status to false");
+      options.isPublic = false;
+    }
```

Comment:
Let's make sure only one of the two is used at once.

Reply:
That's in the parameter validation